### ISSUE TITLE
New installation failing

### ISSUE THREAD

**Issue by pe1chl** [thatmattlove/hyperglass#216]
I am trying to install Hyperglass in a fresh Debian Bullseye VM.  I am not inexperienced but I cannot figure it out.

First I tried the manual installation.  It tells me what dependencies to install but that list is incomplete.
After installing some library development packages like libjpeg-dev libz-dev I was able to get the pip3 install hyperglass to complete, I made a hyperglass.yaml and devices.yaml, but now I am stuck in the hyperglass build-ui phase.
It errors out with:

Errors:
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

However when I run that command it says: 
Cannot find package.json. Is this the right directory to run `npx update-browserslist-db` in?

Probably not.... but what is the right directory?  Maybe look where that package.json file is hiding:
find . -name package.json|wc -l
2376

Ok, 2376 directories with a package.json.  But which one is the correct one?

Then I tried the automatic install (which I of course skipped first, being wary of running a shell script as root).  It errors out with:
[INFO] Installing hyperglass...
  File "/tmp/get-poetry.py", line 1
    404: Not Found
    ^
SyntaxError: illegal target for annotation

Apparently the site where that file is pulled from no longer hosts it.

How to proceed?

**Comment by cogenyk** [comment]
poetry install changed to https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py

**Comment by pe1chl** [comment]
I tried running that.  At first it complained about missing python3-venv, I installed that and tried again.
Now it prints:

python3 install-poetry.py 
The canonical source for Poetry's installation script is now https://install.python-poetry.org. Please update your usage to reflect this.
Retrieving Poetry metadata

# Welcome to Poetry!

This will download and install the latest version of Poetry,
a dependency and package manager for Python.

It will add the `poetry` command to Poetry's bin directory, located at:

/home/scoop/.local/bin

You can uninstall at any time by executing this script with the --uninstall option,
and these changes will be reverted.

Installing Poetry (1.3.2): Done

Poetry (1.3.2) is installed now. Great!

You can test that everything is set up by executing:

`poetry --version`

That does not seem right.

**Comment by pe1chl** [comment]
Ok of course that is only one step in the total process.
I will need to go back to square one.  Probably best to first remove and re-install the VM.

**Comment by pe1chl** [comment]
Running it as root, it cannot find poetry which it is supposed to have installed.
Running it as a user, it complains it has to run as root.

I am bailing out.  Installing Python software is not for me!

**Comment by cogenyk** [comment]
I’d try running a lower LTS maybe 20.04 as it’s supported till 20125 and should support the packages out of the box with a manual install

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pe1chl** [comment]
I have given up and deleted the VM.  Will look for another solution.
Even when running this "fully automatic install" it failed at first because it requires "git" to be installed, but it did not install it by itself.  I did that manually, but then I ran into the other problems.
I think this method of installation is only working on the author's machine or an exact copy, it will not work on anything else.  Specified prerequisites are incomplete, scripts are untested and not working, user-id handling is a mess.
Again, not for me, I will avoid python based projects again for quite some time!
